lib/srv/db/cloud/iam.go

```diff
@@ -61,31 +98,57 @@ func NewIAM(ctx context.Context, config IAMConfig) (*IAM, error) {
 		return nil, trace.Wrap(err)
 	}
 	return &IAM{
-		cfg: config,
-		log: logrus.WithField(trace.Component, "iam"),
+		cfg:   config,
+		log:   logrus.WithField(trace.Component, "iam"),
+		tasks: make(chan iamTask, defaultIAMTaskQueueSize),
 	}, nil
 }
 
+// Start starts the IAM configurator service.
+func (c *IAM) Start(ctx context.Context) error {
+	// DELETE IN 11.0.
+	c.deleteOldPolicy(ctx)
```

Comment:
I think this still deletes the policy right away, no?

Reply:
Just updated. Will wait 10 minutes and checks if new policy is created.